cms/djangoapps/contentstore/features/course-updates.feature

```diff
@@ -2,46 +2,6 @@
 Feature: CMS.Course updates
     As a course author, I want to be able to provide updates to my students
 
-# Commenting out as flaky TNL-5051 07/20/2016
-    # Internet explorer can't select all so the update appears weirdly
-#   @skip_internetexplorer
-#   Scenario: Users can add updates
-#       Given I have opened a new course in Studio
```

Comment:
Delete any unused methods in course-updates.py (the Python file that has the code for these BDD steps).